script.js

```diff
@@ -0,0 +1,8 @@
+function sum()
+{
+    var x = parseInt(document.getElementById("varx").value);
+    var y = parseInt(document.getElementById("vary").value);
+    var result =x+y;
+    document.getElementById("varresult").value = result;
+
```

Comment:
You have an extra line of whitespace here that could be removed. Not a big deal, just good practice.